<launch>
  <node name="rigid_transform_publisher" pkg="core" type="rigid_transform_publisher.py" />
  <node name="rigid_transform_listener" pkg="core" type="rigid_transform_listener.py" />
</launch>
